org.zimwiki.Zim.json

```diff
@@ -0,0 +1,84 @@
+{
+    "app-id": "org.zim.Zim",
```

Comment:
The upstream app ID is org.zim-wiki.Zim and [there's a PR ](https://github.com/jaap-karssenberg/zim-desktop-wiki/pull/307)to change to org.zim_wiki.Zim which is flatpak compatible.

All your exports (e.g. desktop files, icons and appdata etc.) have to have the same app-id naming.

Reply:
Thanks for pointing this out!